.github/workflows/Packages - EdFi.Ods.Minimal.Template.TPDM.PostgreSQL.yml

```diff
@@ -0,0 +1,110 @@
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-TPDM-Artifacts
+          path: Ed-Fi-TPDM-Artifacts/          
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS/
+      shell: powershell
+      run: |
+        .\build.githubactions.ps1 CheckoutBranch -RelativeRepoPath "../Ed-Fi-ODS-Implementation"
+    - name: Install Postgres13
+      shell: pwsh
+      run: |
+          choco install postgresql13 --params '/Password:postgres'
+          $confPath = "C:\Program Files\PostgreSQL\13\data\pg_hba.conf"
+          (Get-Content $confPath).Replace("scram-sha-256","trust") | Set-Content $confPath
+          Restart-Service postgresql-x64-13
+    - name: Initialize-DevelopmentEnvironment
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      run: |
+        $ErrorActionPreference = 'Stop'
+        [Environment]::SetEnvironmentVariable('msbuild_buildConfiguration', '${{ env.CONFIGURATION }}')
+
+        $PSVersionTable
+          . $env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/Initialize-PowershellForDevelopment.ps1
+
+        Initialize-DevelopmentEnvironment -UsePlugins -NoDeploy -Engine PostgreSQL
+      shell: powershell
+    - name: Create Database Template (no extensions)
+      working-directory: ./
+      run: |
+        $ErrorActionPreference = 'Stop'
+        [Environment]::SetEnvironmentVariable('msbuild_buildConfiguration', '${{ env.CONFIGURATION }}')
+        Import-Module -Force -Scope Global "$env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/DatabaseTemplate/Modules/create-tpdm-minimal-template.psm1"
+        Initialize-TPDMMinimalTemplate -Engine PostgreSQL
+      shell: powershell
+    - name: pack
+      working-directory: ./Ed-Fi-ODS/
+      run: |
+        .\build.githubactions.ps1 pack -Configuration ${{ env.CONFIGURATION }} -InformationalVersion ${{ env.INFORMATIONAL_VERSION }} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -NuspecFilePath "$env:GITHUB_WORKSPACE/Ed-Fi-ODS-Implementation/DatabaseTemplate/Database/EdFi.Ods.Minimal.Template.TPDM.Core.PostgreSQL.nuspec" -PackageName "EdFi.Suite3.Ods.Minimal.Template.TPDM.Core.PostgreSQL"
+      shell: powershell
+    - name: Install-credential-handler
+      run: iex "& { $(irm https://aka.ms/install-artifacts-credprovider.ps1) } -AddNetfx"
+      shell: powershell
+    - name: publish 
+      working-directory: ./Ed-Fi-ODS/
+      run: |
+        .\build.githubactions.ps1 publish -InformationalVersion ${{ env.INFORMATIONAL_VERSION }} -BuildCounter ${{ github.run_number }} -BuildIncrementer ${{env.BUILD_INCREMENTER}} -NuGetApiKey ${{ env.AZURE_ARTIFACT_NUGET_KEY }} -EdFiNuGetFeed ${{env.AZURE_ARTIFACT_URL}} -PackageName  "EdFi.Suite3.Ods.Minimal.Template.TPDM.Core.PostgreSQL"
+      shell: powershell
+    - name: Upload EdFi.Ods.Minimal.Template.TPDM.PostgreSQL Artifacts
```

Comment:
The previous build in TeamCity also had a results.zip file that contained the logs from the Bulk Load Client and from the Test Harness, whether it was a successful or failed build. Should those be added as artifacts of this build as well? This looks to be something we missed with porting over the other template builds as well so if we want to add it, we should create a ticket to add it back to the other template builds here in Github as well.

![image](https://user-images.githubusercontent.com/1013553/197580992-a9a256bc-514a-4c67-87af-0488dc6b19a7.png)

Reply:
![image](https://user-images.githubusercontent.com/34613894/197635973-3fc5734c-7769-4183-adf1-a7c22136015e.png)

Just having it in  NugetPackages.Artifacts zip is enough I think but definitely need logfile file and also testharnesslog file .